### ISSUE TITLE
Falsche Entscheidungen:

### ISSUE THREAD

**Issue by Curlie** [sweIhm-ws2018-19/skillproject-fr-15#47]
Als Spieler möchte ich nach falschen Entscheidungen eine "Strafe" erhalten, um die Spannung zu erhöhen.
Ab einer Auswahl zweier falscher Gegenstände, muss ein Zeitabzug gegeben sein.

**Comment by Curlie** [comment]
Funktionalität ist umgesetzt. Sobald die definierte Anzahl an Fehlversuchen überschritten wird, wird das Spiel beendet.

**Comment by Lenz-K** [comment]
- Ich würde die Anzahl an maximalen Versuchen noch erhöhen. Oder diese vielleicht sogar über Schwierigkeitsgrade variabel machen. Aber das ist dann wahrscheinlich eher was für Sprint drei.
- Wenn man verliert muss auch der Gamestate auf Menu gesetzt werden oder!? Und wo wird der Gamestate von Finished auf Menu gesetzt?

**Comment by Curlie** [comment]
Ja, das war auch schon eine Überlegung, die Anzahl kann über die Konstante MAXATTEMPTS in Constant variiert werden.
Nein, sowohl bei verloren als auch gewonnen, wird der Status auf Finished gesetzt. Dann wird Daniels (bald existenter) Intenthandler aufgerufen und darin werden dann Fehlversuche bzw. Gewinne verglichen usw. und dann wird der Status auf MENU zurückgesetzt. So istjedenfalls der Plan.
